_source/_posts/2021-08-11-okta-identity-engine-early-access-hackathon.md

```diff
@@ -0,0 +1,29 @@
+# Join the Okta Identity Engine Early Access Hackathon 2021 Challenge
+
+*Amaze us! Build something unique and wonderful with early access to our new Okta Identity Engine Limited GA release. Here’s how you can get involved.* 
+
+You’re invited to participate in the Okta Identity Early Access Hackathon, which is happening now! Our virtual hackathon began on August 3 and will run till September 14, 2021. You still have more than a month to take the new Okta Identity Engine (OIE) out for a spin and share your skills. We’ll have a total of USD $18,000 in prizes to distribute to the winners! Plus, cool swag prizes for our most supportive and collaborative hackers.  
```

Comment:
I can't help but think this should have a snippet about what OIE is and why it's better than Classic.

Reply:
Based on the advocacy team conversation yesterday, I think it makes most sense in a post of this scope to keep it really simple (and not political). Instead, I added a pointer to the OIE documentation homepage and focused on a mention of new "features" that participants could explore while building something new. It's a little further in (2nd para in the "Rules, Req..." section).